Docs.Site/Docs/Release-Notes/MyGet-2017.2.markdown

```diff
@@ -0,0 +1,123 @@
+# MyGet 2017.2 Release Notes
+
+MyGet 2017.2 was released on **TODO**, 2017.
+
+## Highlights
+
+Next to some new [features](#Features) and many [fixes](#Bug_Fixes_amp_Other_Improvements), this 2017.2 release of MyGet again adds some new functionality to the service.
```

Comment:
Does that navigation to "fixes" work? Seems strange format (the `amp`)

Reply:
Yep it does :)